test/functional/services/data_grid.ts

```diff
@@ -174,7 +174,8 @@ export class DataGridService extends FtrService {
 
     const textArr = [];
     for (const cell of result) {
-      textArr.push(await cell.getVisibleText());
```

Comment:
@Dmitriynj this cause the tests to fail since due to the rendering of the callout, the text was no longer visible ... on the long term it's preferable to hide such callouts for the functionals, since if there's a failure screens just would show the callout.